{"error": null, "issues": [{"address": 427, "code": "        // Track ETH ", "debug": "", "description": "This contract executes a message call to to another contract. Make sure that the called contract is trusted and does not execute user-supplied code.", "filename": "/unique_chucks/31/0x1ff826b46424033d54e5c9ef092ac575604f9295.sol", "function": "fallback", "lineno": 91, "title": "Message call to external contract", "type": "Informational"}, {"address": 427, "code": "        // Track ETH ", "debug": "", "description": "Multiple sends exist in one transaction, try to isolate each external call into its own transaction. As external calls can fail accidentally or deliberately.\nConsecutive calls: \nCall at address: 1004\nCall at address: 899\nCall at address: 697\n", "filename": "/unique_chucks/31/0x1ff826b46424033d54e5c9ef092ac575604f9295.sol", "function": "fallback", "lineno": 91, "title": "Multiple Calls", "type": "Information"}, {"address": 697, "code": "funds += msg.value;\n        \n     ", "debug": "", "description": "This contract executes a message call to to another contract. Make sure that the called contract is trusted and does not execute user-supplied code.", "filename": "/unique_chucks/31/0x1ff826b46424033d54e5c9ef092ac575604f9295.sol", "function": "fallback", "lineno": 92, "title": "Message call to external contract", "type": "Informational"}, {"address": 697, "code": "funds += msg.value;\n        \n     ", "debug": "", "description": "Multiple sends exist in one transaction, try to isolate each external call into its own transaction. As external calls can fail accidentally or deliberately.\nConsecutive calls: \nCall at address: 1004\nCall at address: 899\n", "filename": "/unique_chucks/31/0x1ff826b46424033d54e5c9ef092ac575604f9295.sol", "function": "fallback", "lineno": 92, "title": "Multiple Calls", "type": "Information"}, {"address": 727, "code": "balance / 100) * s", "debug": "storage_4: 0xb0891dba239fe09cdf200020067ecb3fc98f8350bf0c7d0b5f5ae0a3ff88a02e\nretval_427: 0xccb1b0001fc8fe677ec093e5dbd95ffffbfdff3801bf1bf7fc4bfffdbf377f30\ncallvalue: 0x5aa822f71d9ce884f9bf8e99cb2070c0578969a5c97ecb7120f00f1943686083\nretval_697: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nreturndatasize: 0x20\nextcodesize: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nstorage_5: 0x0\ncalldata_PearlBonus_0: 0xf00000000000000000000000000000000000000000000000000000000\ncalldatasize_PearlBonus: 0x4\n", "description": "A possible integer overflow exists in the function `fallback`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/unique_chucks/31/0x1ff826b46424033d54e5c9ef092ac575604f9295.sol", "function": "fallback", "lineno": 95, "title": "Integer Overflow ", "type": "Warning"}, {"address": 733, "code": "balance / 100) * s", "debug": "", "description": "The contract account state is changed after an external call. Consider that the called contract could re-enter the function before this state change takes place. This can lead to business logic vulnerabilities.", "filename": "/unique_chucks/31/0x1ff826b46424033d54e5c9ef092ac575604f9295.sol", "function": "fallback", "lineno": 95, "title": "State change after external call", "type": "Warning"}, {"address": 899, "debug": "", "description": "Multiple sends exist in one transaction, try to isolate each external call into its own transaction. As external calls can fail accidentally or deliberately.\nConsecutive calls: \nCall at address: 1004\n", "filename": "/unique_chucks/31/0x1ff826b46424033d54e5c9ef092ac575604f9295.sol", "function": "fallback", "lineno": 99, "title": "Multiple Calls", "type": "Information"}, {"address": 899, "debug": "", "description": "A possible transaction order independence vulnerability exists in function fallback. The value or direction of the call statement is determined from a tainted storage location", "filename": "/unique_chucks/31/0x1ff826b46424033d54e5c9ef092ac575604f9295.sol", "function": "fallback", "lineno": 99, "title": "Transaction order dependence", "type": "Warning"}, {"address": 2044, "code": "tion\n     */\n    function () pu", "debug": "", "description": "A possible transaction order independence vulnerability exists in function withdrawFunds(). The value or direction of the call statement is determined from a tainted storage location", "filename": "/unique_chucks/31/0x1ff826b46424033d54e5c9ef092ac575604f9295.sol", "function": "withdrawFunds()", "lineno": 75, "title": "Transaction order dependence", "type": "Warning"}, {"address": 2455, "code": "* Transfers the director to a ", "debug": "", "description": "This contract executes a message call to to another contract. Make sure that the called contract is trusted and does not execute user-supplied code.", "filename": "/unique_chucks/31/0x1ff826b46424033d54e5c9ef092ac575604f9295.sol", "function": "transfer(address,uint256)", "lineno": 61, "title": "Message call to external contract", "type": "Informational"}], "success": true}